File: basics/trail.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
    freopen("input.txt","r",stdin);
    freopen("output.txt","w",stdout);
    map<int,int> m;
    int n=3;
    int arr[]={10,40,20};
    for(int j=0;j<n;j++)
{
    m[arr[j]]=j;
}
    map<int,int>::iterator i;
    for(i=m.begin();i!=m.end();i++)
    cout<<i->first<<"    "<<i->second<<endl;
    int j=0;
    for(i=m.begin();i!=m.end();i++)
    {
arr[i->second]=j;
j++;
    }
    for(j=0;j<n;j++)
    cout<<arr[j]<<" ";

}

File: fst software's/Programming related/trail.py
n=input()
k=input()
n=list(set(list(n)))
k=list(set(list(k)))
v=0
u=0
for i in n:
    if i not in k:
        v=1
        break
for i in k:
    if i not in n:
        u=1
        break
print(u==0 and v==0)
